### PR TITLE
Hotfix/python signature

### PR DESCRIPTION
**Pull request by fkaufman-asym**
Need to add environment variable so we don't have to keep changing the endpoint.

### CODE REVIEW

**Comment by fkaufman-asym** [comment]
The system parameter needs to be an actual string to match the generated OpenAPI python client interface

File: cdapython.py [hunk 1]
```diff
@@ -161,14 +161,14 @@ def columns(version=table_version, host=CDA_API_URL):
...
 
-def unique_terms(col_name, system=None, version=table_version, host=CDA_API_URL):
+def unique_terms(col_name, system=''):
     with cda_client.ApiClient(
-            configuration=cda_client.Configuration(host=host)
+            configuration=cda_client.Configuration(host=CDA_API_URL)
     ) as api_client:
         api_instance = QueryApi(api_client)
-
+        api_response = api_instance.unique_values(version=table_version, body=col_name, system=system)
```
Comment (dionboles-asym) [inline cdapython.py:169]: Why not cast the system param to a string you can use str() to make sure it is a str so i will not fail in a post to the server?

**Review by dionboles-asym — COMMENTED** [review]
Good fix

File: cdapython.py (hunk 1 above)
Comment (nemanjab17) [reply cdapython.py:170]: also, pass it directly to the unique_values method as str(system), no need to assign variable if it is referenced only once below

**Pull request merged**